components/style/properties/longhand/outline.mako.rs

```diff
@@ -58,4 +58,10 @@ ${helpers.predefined_type("outline-color", "CSSColor", "::cssparser::Color::Curr
     }
 </%helpers:longhand>
 
+% for corner in ["topleft", "topright", "bottomright", "bottomleft"]:
```

Comment:
I'd add a comment indicating that this property is deprecated and not on a standards track (which is my reading of the situation).